src/vumi2/routers.py

```diff
@@ -49,6 +50,15 @@ async def setup(self):
                 connector_name=name, outbound_handler=self.handle_outbound_message
             )
 
+        if (
+            self.config.default_app
+            and self.config.default_app not in self.receive_outbound_connectors
+        ):
+            self.default_connector = await self.setup_receive_outbound_connector(
```

Comment:
I don't see us using `self.default_connector` anywhere. Should we maybe skip the assign?